public/locations-list-map.js

```diff
@@ -1,6 +1,7 @@
 import toDataByLocation from './toDataByLocation.js';
 
 function createFiltersListHTML() {
+  $('.locations-loading').hide();
```

Comment:
At this point it's possible that the map will still not have loaded. This function `createFiltersListHTML` is just about creating the state filters and not related to the map loading.

I think there's something you can do where you add an event listener on the `tilesloaded` event from Google Maps, which should tell you more accurately when the map has loaded (see this [stackoverflow post](https://stackoverflow.com/questions/33234945/how-to-detect-if-google-map-loaded-successfully)). I was able to reproduce this in the browser console, so I think it should work!

Reply:
That's true, but there are a few reasons why I went this way (and keep in mind that I'm still conforming to the exiting JavaScript structure).

The JS is structured in a certain way the number one important goal here is to ensure it's as intuitive as possible (i.e. hide one thing and show the other, they're contextually relevant albeit it's not 100% accurate in terms of actual execution vs. intent). In this case placing it here for the most obvious place to find it, helps adhere to [POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).


Given this site is subject to VERY rapid change, I'm conforming to the existing structure. If I built it I may restructure it and move **both** the `hide()`/`show()` methods OUT of the `createFiltersListHTML()` function because the name _implies_ that it's only job is to "create HTML", but that's not all it does. So, in the interest of speed I just rolled w/ what's already here.

Make sense?

Reply:
Here you go, check this out  @akainic: 9c1db55 Thanks for the guidance :)

Reply:
Thanks for these thoughts! I just took a look at the new commit. My concern is that it doesn't quite solve the problem in the original issue, which is that we're not showing anything while we wait for the map to load. If we aren't making `$('.locations-container').show();` dependent on whether the map has actually loaded, do we have confidence that it has? I was just suggesting adding an event listener and conditionally doing `$('.locations-container').show();` based on the when the listener is fired. I think this would be fairly simple, but if it's too much to do in this PR I can try to fix this later!